eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -104,6 +105,93 @@ initiateHardForkWithLessThanMinimalCommitteeSize =
         passNEpochs 2
         getLastEnactedHardForkInitiation `shouldReturn` SNothing
 
+spoAndCCVotingSpec ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+spoAndCCVotingSpec = do
+  describe "When CC expired" $ do
+    let setup = do
+          hotCs <- registerInitialCommittee
+          -- TODO: change the maxtermlength to make the test faster
+          EpochInterval committeeMaxTermLength <-
+            getsNES $
+              nesEsL . curPParamsEpochStateL . ppCommitteeMaxTermLengthL
+          passNEpochs $ fromIntegral committeeMaxTermLength
+          ms <- getCommitteeMembers
+          -- Make sure that committee expired
+          forM_ ms ccShouldBeExpired
+          pure hotCs
+    it "SPOs alone can't enact hard-fork" $ do
+      hotCs <- setup
+      (spoC, _, _) <- setupPoolWithStake $ Coin 1_000_000_000
+      protVer <- getProtVer
+
+      gai <- submitGovAction $ HardForkInitiation SNothing (majorFollow protVer)
+
+      submitYesVote_ (StakePoolVoter spoC) gai
+      -- CC members expired so their votes don't count - we are stuck!
+      submitYesVoteCCs_ hotCs gai
+
+      passNEpochs 2
+
+      getLastEnactedHardForkInitiation `shouldReturn` SNothing
+    it "SPOs alone can't enact security group parameter change" $ do
+      void setup
+      (spoC, _, _) <- setupPoolWithStake $ Coin 1_000_000_000
+
+      gid <-
+        submitParameterChange SNothing $
+          emptyPParamsUpdate
+            & ppuMinFeeRefScriptCostPerByteL .~ SJust (25 %! 2)
+
+      submitYesVote_ (StakePoolVoter spoC) gid
+
+      passNEpochs 2
+
+      getLastEnactedParameterChange `shouldReturn` SNothing
+  describe "When CC threshold is 0" $ do
+    let
+      modifyCommittee f = modifyNES $ \nes ->
+        nes
+          & newEpochStateGovStateL . committeeGovStateL %~ f
+          & newEpochStateDRepPulsingStateL %~ modifyDRepPulser
+        where
+          modifyDRepPulser pulser =
```

Comment:
Why this? What is the effect of modifying the state in this way?  Would be nice if we could replace it via normal operations.  
Maybe at least maybe add a comment or better yet an assertion,  to specify what premise we are creating with this change.

Reply:
During bootstrap, proposals that modify the committee are not allowed so this is the only way we can change their threshold.